### PR TITLE
Revise documentation around tools

### LINKED ISSUE #967: Revise documentation around tools
Currently [here](https://github.com/hashgraph/hedera-json-rpc-relay/tree/main/tools#option-1) we are saying that using ethersjs and hardhat, we need to re-init the contract in order to be able to use it.
This is the old way, which we needed in order to be able to use the contract:
```
// init the contract factory
const factory = new ethers.ContractFactory(contractJson.abi, contractJson.bytecode, wallet);
// deploy the contract
let contract = await factory.deploy();

// wait till the transaction has reached consensus and get the contract address from the receipt
const { contractAddress } = await contract.deployTransaction.wait();

// re-init the contract with the deployed address
contract = new ethers.Contract(contractAddress, contractJson.abi, wallet);
```

We can now do this:
```
// init the contract factory
const factory = new ethers.ContractFactory(contractJson.abi, contractJson.bytecode, wallet);
// deploy the contract
let contract = await factory.deploy();

// wait till the transaction has reached consensus and get the contract address from the receipt
const { contractAddress } = await contract.deployTransaction.wait();

//use contract without re-init
await contract.someFunction();
```

### PR DESCRIPTION
**Pull request by georgi-l95**
**Description**:
Removes the notes about using and deploying contracts with ethers and hardhat, because we no longer need to re init the contract, to be able to use it. So experience should be the same as Ethereum.

This is the old way, which we needed in order to be able to use the contract:
```
// init the contract factory
const factory = new ethers.ContractFactory(contractJson.abi, contractJson.bytecode, wallet);
// deploy the contract
let contract = await factory.deploy();

// wait till the transaction has reached consensus and get the contract address from the receipt
const { contractAddress } = await contract.deployTransaction.wait();

// re-init the contract with the deployed address
contract = new ethers.Contract(contractAddress, contractJson.abi, wallet);
```

We can now do this:
```
// init the contract factory
const factory = new ethers.ContractFactory(contractJson.abi, contractJson.bytecode, wallet);
// deploy the contract
let contract = await factory.deploy();

// wait till the transaction has reached consensus and get the contract address from the receipt
const { contractAddress } = await contract.deployTransaction.wait();

//use contract without re-init
await contract.someFunction();
```

Also I've bumped the mirror and network node images.

**Related issue(s)**:

Fixes #967 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by AlfredoG87 — APPROVED** [review]
LGTM 👍

**Review by lukelee-sl — APPROVED** [review]
LG

File: packages/server/tests/acceptance/index.spec.ts [hunk 1]
```diff
@@ -132,17 +132,17 @@ describe('RPC Server Acceptance Tests', function () {
...
+        process.env['MIRROR_IMAGE_TAG'] = '0.76.1';
 
         console.log(`Docker container versions, services: ${process.env['NETWORK_NODE_IMAGE_TAG']}, mirror: ${process.env['MIRROR_IMAGE_TAG']}`);
 
         console.log('Installing local node...');
         shell.exec(`npm install @hashgraph/hedera-local -g`);
 
         console.log('Starting local node...');
-        shell.exec(`hedera start -d`);
+        shell.exec(`hedera start -d --turbo`);
```
Comment (ebadiere) [inline packages/server/tests/acceptance/index.spec.ts]: Perhaps not relevant to this change.  I see the option --turbo is listed in the README as "to enable/disable turbo mode."  Is turbo mode described in more detail somewhere.  I assume turbo means the local node runs faster, but what is being sacrificed to get the speed?

**Review by Nana-EC — APPROVED** [review]

File: packages/server/tests/acceptance/index.spec.ts (hunk 1 above)
Comment (georgi-l95) [reply packages/server/tests/acceptance/index.spec.ts]: Yes, it refers to a faster mode in the local-node. No sacrifice, we just skip the usage of minio and uploaders. This saves some time on exporting and importing record files. We have a issue in the local-node to add docs with diagrams about this change, I will soon start working on it.

**Pull request merged**